### PR TITLE
Add metrics documentation

### PR DESCRIPTION
**Pull request by saschagrunert**
Add available metrics, their labels and the purpose in a dedicated
document. This documents is now linked in the main README.md as well.

### CODE REVIEW

File: tutorials/metrics.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+endpoint manually via [curl][1].
+
+```bash
+> curl localhost:9090/metrics
+…
+```
+
+## Available Metrics
+
+Beside the [default golang based metrics][2], CRI-O provides the following additional metrics too:
```
Comment (TomSweeneyRedHat) [inline tutorials/metrics.md]: I'd drop 'too' here also.    Just end with 'metrics:'

File: tutorials/metrics.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+| `crio_operations`                      | every CRI-O RPC name: `Attach`, `ContainerStats`, `ContainerStatus`, `CreateContainer`, `Exec`, `ExecSync`, `ImageFsInfo`, `ImageStatus`, `ListContainerStats`, `ListContainers`, `ListImages`, `ListPodSandbox`, `PodSandboxStatus`, `PortForward`, `PullImage`, `RemoveContainer`, `RemoveImage`, `RemovePodSandbox`, `ReopenContainerLog`, `RunPodSandbox`, `StartContainer`, `Status`, `StopContainer`, `StopPodSandbox`, `UpdateContainerResources`, `UpdateRuntimeConfig`, `Version` | Counter | Cumulative number of CRI-O operations by operation type.                 |
```
Comment (TomSweeneyRedHat) [inline tutorials/metrics.md]: Side question.  I just went looking quickly, are these RPC's documented anywhere?  It seems like they should have a page of their own listing each one with a brief description of how to call it and what it does.

File: tutorials/metrics.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+| `crio_operations_latency_microseconds` | _every CRI-O RPC name_,<br>`network_setup_pod`: CNI pod network setup time,<br>`network_setup_overall`: Overall network setup time                                                                                                                                                                                                                                                                                                                                                         | Summary | Latency in microseconds of CRI-O operations. Split-up by operation type. |
```
Comment (TomSweeneyRedHat) [inline tutorials/metrics.md]: So I think you're saying here is that every CRI-O RPC name can be used to get metrics for crio_operations, then for the crio_operations_* ones the CRI-O RPC's  plus the other listed items can be used.   If so, then this is fine, although to make it easier to read, just link "Every CRI-O RPC" to a list elsewhere on the page that would look like:

```
CRI-O RPC's: `Attach`, `ContainerStats`, `ContainerStatus`, `CreateContainer`, `Exec`, `ExecSync`, `ImageFsInfo`, `ImageStatus`, `ListContainerStats`, `ListContainers`, `ListImages`, `ListPodSandbox`, `PodSandboxStatus`, `PortForward`, `PullImage`, `RemoveContainer`, `RemoveImage`, `RemovePodSandbox`, `ReopenContainerLog`, `RunPodSandbox`, `StartContainer`, `Status`, `StopContainer`, `StopPodSandbox`, `UpdateContainerResources`, `UpdateRuntimeConfig`, `Version`
```

Or better yet, to a page documenting the RPC's.

File: tutorials/metrics.md (hunk 1 above)
Comment (saschagrunert) [reply tutorials/metrics.md]: Yep, fixed :+1:

File: tutorials/metrics.md (hunk 2 above)
Comment (saschagrunert) [reply tutorials/metrics.md]: Yes, I added a link below the table.

File: tutorials/metrics.md (hunk 3 above)
Comment (saschagrunert) [reply tutorials/metrics.md]: I added the RPCs below the table to make it easier to read, WDYT?

File: tutorials/metrics.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply tutorials/metrics.md]: 'too:' looks to have survived....

File: tutorials/metrics.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+| `crio_operations_errors`               | every CRI-O RPC\*                                                                                                                       | Counter | Cumulative number of CRI-O operation errors by operation type.           |
+
+\* Available CRI-O RPC's from the [gRPC API][3]: `Attach`, `ContainerStats`, `ContainerStatus`,
+`CreateContainer`, `Exec`, `ExecSync`, `ImageFsInfo`, `ImageStatus`,
+`ListContainerStats`, `ListContainers`, `ListImages`, `ListPodSandbox`,
+`PodSandboxStatus`, `PortForward`, `PullImage`, `RemoveContainer`,
+`RemoveImage`, `RemovePodSandbox`, `ReopenContainerLog`, `RunPodSandbox`,
+`StartContainer`, `Status`, `StopContainer`, `StopPodSandbox`,
+`UpdateContainerResources`, `UpdateRuntimeConfig`, `Version`
+
```
Comment (TomSweeneyRedHat) [inline tutorials/metrics.md]: Thanks for breaking this out, I think the table is a lot more readable now.

File: tutorials/metrics.md (hunk 1 above)
Comment (saschagrunert) [reply tutorials/metrics.md]: Ah, fixed (finally)

**Comment by haircommander** [comment]
LGTM
not much point in waiting on tests to pass for a docs only pr. merging manually

**Pull request merged**